operator/docs/lokistack/sop.md

```diff
@@ -34,8 +34,8 @@ A service(s) is failing to process at least 10% of all incoming requests.
 - Console access to the cluster
 - Edit access to the deployed operator and Loki namespace:
   - OpenShift
-    - `openshift-logging`
-    - `openshift-operators-redhat`
+    - `openshift-logging` (Lokistack)
```

Comment:
I think we should choose one notation for `LokiStack` that we use throughout the docs. Currently it seems `LokiStack` is used more than `Lokistack`.

```suggestion
    - `openshift-logging` (LokiStack)
```